src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -1382,7 +1387,7 @@ private static boolean validateConfigFile(String file, CType cType, int version)
 
 	private static String[] getTypes(boolean legacy) {
 		if (legacy) {
-			return new String[]{"config", "roles", "rolesmapping", "internalusers", "actiongroups", "nodesdn", "audit"};
+			return new String[]{"config", "roles", "rolesmapping", "internalusers", "actiongroups", "nodesdn", "audit","tenancyconfig"};
```

Comment:
This legacy block is only for CType for ES6. This can be removed from this block.

Reply:
Thanks for pointing this out @cwperks. Will remove this in next commit.